storage/innobase/log/log0recv.cc

```diff
@@ -1107,8 +1113,11 @@ struct Log_checkpoint_location {
 @param[in,out]  log             redo log
 @param[out]     checkpoint      the latest checkpoint found (if any)
 @return true iff any checkpoint has been found */
-static bool recv_find_max_checkpoint(log_t &log,
-                                     Log_checkpoint_location &checkpoint) {
+#ifndef XTRABACKUP
+static
+#endif /*XTRABACKUP */
```

Comment:
may be its time to create IF_N_XB() similar to IF_XB(). Then we can use readable code
IF_N_XB(static)
bool recv_find_max_checkpoint()..

Reply:
Should I create a bug to fix it? so we can fix other occurrence as well.

Reply:
keep it as is for now. No need to creat IF_N_XB(). we will do it next time. Droping this